osmUtils/osmDownload.py

```diff
@@ -92,3 +94,43 @@ def get_osm_gdf(self):
         """
         gdf = generate_osm_gdf(response_json=self.osm_json)
         return gdf
+
+class SavetoFile:
+    """
+        Save response geogapdas.GeoDataFrame to local file
+        
+        Parameters
+        ----------
+        gdf: geopandas.GeoDataFrame
+            response from overpass API in geopandas.GeoDataFrame format
+        output_path: string
+            File path or file handle to write to.
+        driver : string, default: 'ESRI Shapefile'
+            The OGR format driver used to write the vector file.
+        """
+    def __init__(self ,gdf, filename='./osm_data.shp', driver="ESRI Shapefile", **kwargs):
+        self.gdf = gdf
+        self.filename = filename
+        self.driver = driver
+        self.kwargs = kwargs
+        self.save = self.save_gdf_to_file()
```

Comment:
So it always saves to file on initialisation? I think this may be a case where creating a class is unnecessary, let's have a chat :)